README.md

```diff
@@ -31,5 +40,30 @@ too; replace `-full` with `-oss` in any of the above commands to install the
 OSS distribution. Note that the default distribution and OSS distribution of
 a product can not be installed at the same time.
 
+## How do I ensure my configuration is up to date?
+
+Run the following command to update your configuration:
+
+    brew tap --repair
```

Comment:
It could be best to only repair our tap, to help avoid causing issues for anything else on folks systems
```suggestion
    brew tap --repair elastic/tap
```